main.go

```diff
@@ -101,15 +106,96 @@ func validateMongoURL(mongourl string) error {
 	return nil
 }
 
+func getSecret(secretName string , region string) string {
```

Comment:
It will be more idiomatic to return `(string, error)` from the `getSecret` function and let the error be handled by the caller instead of printing the error.